pkgs/tools/admin/balena-cli/default.nix

```diff
@@ -16,13 +16,13 @@ let
   }.${system} or throwSystem;
 
   sha256 = {
-    x86_64-linux = "0gxki6w8p7ihv0zy02978hg8i242algiw0wpcajrvbx1ncbcb7yn";
+    x86_64-linux = "sha256-5Wj68stbxlKu+68AjnC9EePBYJ6Ga/FOvdL0Jt5V7xA=";
     x86_64-darwin = "1ihxyf35px3s6q2yk4p3dy03rcj93hy96bj3pxqlv0rp05gnsf02";
     aarch64-darwin = "1ihxyf35px3s6q2yk4p3dy03rcj93hy96bj3pxqlv0rp05gnsf02";
     x86_64-windows = "104hc3qvs04l2hmjmp0bcjr5g5scp4frhprk1fpszziqhdmhwa40";
```

Comment:
What is the best way to add these hashes? May I push to this branch with the updates?

```
x86_64-darwin = "sha256-eW6WtS4BMvQDJ9zEJWniXZVqfHPD2N0NZt90dBfhxBI=";
aarch64-darwin = "sha256-eW6WtS4BMvQDJ9zEJWniXZVqfHPD2N0NZt90dBfhxBI=";
x86_64-windows = "sha256-ociIIxtZ4v7mSedwPvRzELrj1OEddcdyNEOYeBL8iIg=";
```